core/src/main/resources/schema/app/20181111232339_BUG-113430_Support_KDC_with_self_signed_cert.sql

```diff
@@ -0,0 +1,9 @@
+-- // BUG-113430_Support_KDC_with_self_signed_cert
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE kerberosconfig ADD COLUMN IF NOT EXISTS verifykdctrust BOOLEAN DEFAULT false;
```

Comment:
for existing clusters I don't know if `true` or `false` is the better. Now I think `true` would be better.